Integrations/SecBI/SecBI.py

```diff
@@ -0,0 +1,197 @@
+                                                                                         SECBI_API_ENDPOINT_INCIDENTS,
+                                                                                         str(params)))
+        try:
+            ids_list_str = self._http_request(method='GET',
+                                              url_suffix=SECBI_API_ENDPOINT_INCIDENTS,
+                                              params=params,
+                                              resp_type='text')
+        except Exception as e:
+            LOG("Error when fetching Incidents list from SecBI: {}".format(e.args[0]))
+            LOG.print_log()
+            raise
+        return json.loads(ids_list_str)
+
+    def secbi_get_incident(self, incident_id):
+        """
+        Get a specific SecBI Incident by SecBI Incident ID
+        :param incident_id: SecBI incident ID
+        :return: A dictionary representation of the SecBI Incident requested
+        """
+        params = get_dict_without_none(incidentId=incident_id, fields=SECBI_INCIDENT_FIELDS)
+        try:
+            raw_incident = self._http_request(method='GET',
+                                              url_suffix='{}/{}'.format(SECBI_API_ENDPOINT_INCIDENT, incident_id),
+                                              params=params)
+            secbi_incident = SecBIIncident(raw_incident)
+            return vars(secbi_incident)
+        except Exception as e:
+            LOG("Error when fetching Incident {} from SecBI: {}".format(incident_id, e.args[0]))
+            LOG.print_log()
+            raise
+
+    def secbi_get_incident_by_host(self, host):
+        """
+        Get a specific SecBI Incident by Host
+        :param host: The host by which to get a SecBI Incident
+        :return:  A dictionary representation of the SecBI Incident found to contain the supplied host
+        """
+        params = get_dict_without_none(host=host, fields=SECBI_INCIDENT_FIELDS)
+        try:
+            raw_incident = self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_INCIDENT, params=params)
+            secbi_incident = SecBIIncident(raw_incident)
+            return vars(secbi_incident)
+        except Exception as e:
+            LOG("Error when fetching Incident by host '{}' from SecBI: {}".format(host, e.args[0]))
+            LOG.print_log()
+            raise
+
+    def test_module(self, api_url, api_key):
+        """
+        Performs basic get request to test SecBI system
+        """
+        try:
+            self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_STATUS)
+        except Exception as e:
+            return_error("Failed sanity test of SecBI with SECBI_API_URL '{}', and SECBI_API_KEY '{}': {}".format(api_url, api_key, e.args[0]))
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def get_human_readable_incidents_list(incidents_list):
```

Comment:
Don't really see the added value of a function of 2 lines which is called only once, let's just insert those 2 lines to where this function is called.